omod-1.8/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_8/UserController1_8Test.java

```diff
@@ -245,6 +291,17 @@ public void shouldFindUserByUsername() throws Exception {
 	}
 	
 	@Test
+	public void shouldFindUserByUsernameUsingSearchParameter() throws Exception {
+		SimpleObject response = deserialize(handle(newGetRequest(getURI(), new Parameter("username", "admin"))));
```

Comment:
You do not test here that the correct handler was picked up to do the search. I would change the test to for example:
shouldNotFindAdminByUsername and make the mocked handler return empty list when admin is requested.